### ISSUE TITLE
Accuracy is always zero 

### ISSUE THREAD

**Issue by zeynepgokce** [hx173149/C3D-tensorflow#56]
Hi,
I changed some parts of the codes as following to fine tune the moodle but i am getting error. all the time accuracy is 0. Can you help me ? What is your suggestion? Thank you.
*****************************************
    init = tf.global_variables_initializer()
    # Create a saver for writing training checkpoints.
    saver = tf.train.Saver(varlist1) # Edit : Add ops to save and restore only varibles without out layer using the name "varlist1"
      

    # Create a session for running Ops on the Graph.
    sess = tf.Session(
                    config=tf.ConfigProto(allow_soft_placement=True)
                    )
    sess.run(init)
    if os.path.isfile(model_filename) and use_pretrained_model:
      saver.restore(sess, model_filename) # EDIT
      print("model is restored")
   *****************************************

The output : 
.
.
.
Training Data Accuracy: 0.00000
Validation Data Accuracy: 0.00000
Step 2241: 7.663 sec
Training Data Accuracy: 0.00000
Validation Data Accuracy: 0.00000
Step 2242: 7.619 sec
Training Data Accuracy: 0.00000
Validation Data Accuracy: 0.00000
Step 2243: 7.686 sec
Training Data Accuracy: 0.00000
Validation Data Accuracy: 0.00000
Step 2244: 8.781 sec
Training Data Accuracy: 0.00000
Validation Data Accuracy: 0.00000

**Comment by zkjqw139** [comment]
try use
        labels          =  tf.one_hot(labels_placeholder, c3d_model.NUM_CLASSES)
        loss            = -tf.reduce_sum(labels*tf.log(tf.clip_by_value(tf.nn.softmax(logit),1e-10,1.0)),1.0)
instead

**Comment by 491506870** [comment]
@zeynepgokce hi, could you tell me how to print the loss, where should i add code? thank you.

**Comment by zeynepgokce** [comment]
@491506870 Hi, i just added "loss" to session simply,  as following 

summary, acc,l = sess.run(
                        [merged, accuracy,loss],
                        feed_dict={images_placeholder: train_images,
                            labels_placeholder: train_labels
                            })
        print ("accuracy: " + "{:.5f}".format(acc))
        print(" Loss : ",l)

**Comment by 491506870** [comment]
@zeynepgokce thank you so much!!! which helps me a lot~~and i think i met the same problem with you, my loss became NAN after several steps, what did you do to solve it?

**Comment by zeynepgokce** [comment]
@491506870 , Problem was related to my own Dataset Labelling. I changed my labelling like starting from 0 to 2 since i have 3 classes.
